spec/brca.txt

```diff
@@ -32,20 +32,21 @@ Description:	"Specimen resulting from biopsy or excision of breast and surroundi
 
 
 EntryElement:	BreastCancerCondition
-Based on:		CancerCondition
+Based on:		PrimaryCancerCondition
 Concept:		SCT#126926005 "Neoplasm of breast (disorder)"
```

Comment:
We should add a TODO here to revisit the value set to ensure it aligns with the new concept of `PrimaryCancerCondition`.

Reply:
Tracked as #256